x/merkledb/intermediate_node_db.go

```diff
@@ -66,12 +67,13 @@ func (db *intermediateNodeDB) onEviction(key path, n *node) error {
 	// and write them to disk. We write a batch of them, rather than
 	// just [n], so that we don't immediately evict and write another
 	// node, because each time this method is called we do a disk write.
-	for writeBatch.Size() < db.evictionBatchSize {
+	for totalSize < db.evictionBatchSize {
```

Comment:
Can we update the comment above to make it clear that we're evicting a certain number of bytes rather than a certain number of nodes? We should have done that in #1947 but guess we missed this